.packaging/Dockerfile.gentoo

```diff
@@ -0,0 +1,29 @@
+FROM gentoo/stage3
+
+RUN echo 'FEATURES="-ipc-sandbox -network-sandbox -pid-sandbox -sandbox -usersandbox -usersync -userfetch -userpriv"' >> /etc/portage/make.conf
+
```

Comment:
One only empty line between sections should be sufficient.

Reply:
done